src/errors.rs

```diff
@@ -11,6 +11,7 @@ custom_error! {pub ProgramError
     Io {source: io::Error} = "IO Error : {:?}",
     Crossterm {source: crossterm::ErrorKind} = "Crossterm Error : {:?}",
     Termimad {source: termimad::Error} = "Termimad Error : {:?}",
+	ClipboardError = "Clipboard handler error: unable to copy to clipboard",
```

Comment:
I decided to not include the actual error, because I didn't want to reimport `anyhow::Error` , and I don't expect clipboard actions to fail ...